azkustodata/kql/query_parameters.go

```diff
@@ -1,70 +1,74 @@
 package kql
 
 import (
-	"github.com/Azure/azure-kusto-go/azkustodata/types"
```

Comment:
Why the rename?
The old name sounds more to point.

Reply:
If you have a "types.Bool" you don't have a bool type, you have a bool value. 
That was my logic, but I can revert it otherwise.

Reply:
not sure I follow.
Those are data types. the do store values, but you code the type behavior.
Suggest to revert (it would also simplify this PR)

Reply:
My mistake - I didn't change any namespace, they were named like this before. 
the types package defines the column types (like types.Bool is a constant that means "bool"), and the value defines a value (like value.Bool is a struct that holds a kusto bool ).

The difference here is now I need a different improt